test/cases/cross-distro.sh

```diff
@@ -110,8 +110,8 @@ ALL_DISTROS=$(find "$REPO_PATH" -name '*.json' -printf '%P\n' | awk -F "." '{ pr
 ALL_EXPECTED_DISTROS=$(echo "$ALL_DISTROS" | grep -E "$PATTERN" | grep -Ev 'beta|stream' | sort)
 # Warning: filter out the remaining distros by matching whole words to avoid matching
 # the value rhel-93 by the pattern rhel-9!
-# If we're running on a RHEL 9.2 osbuild-composer doesn't know anything about 9.3
-# images so the value rhel-9.3 should be treated as unrecognized and error out as
+# If we're running on a RHEL 9.3 osbuild-composer doesn't know anything about 9.4
+# images so the value rhel-9.4 should be treated as unrecognized and error out as
```

Comment:
OTOH why not update this to something like `rhel-X.Y` so that we can avoid updating comment in the future ? WDYT ?